src/sagas.js

```diff
@@ -0,0 +1,9 @@
+import { all } from 'redux-saga/effects';
+
+import { profileSaga } from './profile';
+
+export default function* rootSaga() {
+  yield all([
+    profileSaga(),
```

Comment:
Another touch point between the MFE boilerplate and profile module - like reducers above.